src/inverted-index.js

```diff
@@ -112,13 +118,35 @@ class InvertedIndex {
   search(index, term) {
     return this.indices[index][term] ? this.indices[index][term] : [];
   }
+
+  /**
+   * validateFileNames
+   * Checks if the filename actually exists
+   * @param {Array} filenames
+   * @returns {boolean} status
+   * @memberOf InvertedIndex
+   */
+  validateFileNames(filenames) {
+    let status = true;
+    filenames.forEach((filename) => {
+      if (!Object.keys(this.indices).includes(filename)) {
+        status = false;
+      }
+    });
+    return status;
+  }
 }
 
 Array.prototype.flatten = function flatten() {
   return this.toString().split(',').map(item =>
     item.toLowerCase());
 };
 
+String.prototype.cleanDoc = function cleanDoc() {
+  return this.replace(/[^a-z0-9\s]/gi, '')
+    .toLowerCase()
+    .split(' ');
+};
 if (typeof module === 'object' && module.exports) {
   module.exports = InvertedIndex;
```

Comment:
### UltraCritical

@andela-oakinrele you are expected to use the `es6` method for exporting classes e.g `export class InvertedIndex`. Please refactor.